### PR TITLE
Add buttons with gradient border in different sizes

### PR DESCRIPTION
**Pull request by Lenkly**
In this pull request I styled my buttons with gradient (rainbow) border. 

Screenshot 2020-04-17 at 13 53 41

If they don't look like this in the heroku deployment, I failed again at installing global styles 🙈

### CODE REVIEW

**Comment by meloneminze** [comment]
Amazing 🌈

File: client/src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      #ff0084,
+      #ff00f2,
+      #bb00ff,
+      #7700ff,
+      #006aff,
+      #00ccff,
+      #00ff6a,
+      #59ff00,
+      #fff700
+    );
```
Comment (marwinburesch) [inline client/src/components/Button.js:42]: maybe this helps with compatibility. A different approach with just borders and no extra elements:

- the `9001px` is just a joke, it needs to be half the height ;)
- `linear-gradient(white, white)` is the inner part of the button

```suggestion
  height: ${(props) => buttonSize[props.size].height};
  border: double 4px transparent;
  border-radius: 9001px;
  background-image: linear-gradient(white, white), linear-gradient(to right,
      #fff700,
      #ff9100,
      #ff0084,
      #ff00f2,
      #bb00ff,
      #7700ff,
      #006aff,
      #00ccff,
      #00ff6a,
      #59ff00,
      #fff700);
   background-origin: border-box;
   background-clip: padding-box, border-box;
```

File: client/src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  }
+`;
+
+const buttonSize = {
```
Comment (marwinburesch) [inline client/src/components/Button.js:46]: goot stuff!

File: client/src/components/Button.js (hunk 1 above)
Comment (Lenkly) [reply client/src/components/Button.js]: this worked so much better!

**Pull request merged**